### PR TITLE
prob_given_false is soon req, show in examples

### PR DESCRIPTION
**Pull request by HarvsG**
## Proposed change

Corrects some incorrect `to_state`s and a typo
Although though it is not required in 2022.9 ([home-assistant:current](https://github.com/home-assistant/home-assistant.io)) (it will be in 2020.10) there is *no* mathematical rationale for not having `prob_given_false` in the config otherwise the use invited unexpected and buggy behaviour, therefore this should be reflected in the examples.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
It seems that this PR is targeted against an incorrect branch since it has a parent PR on one of our codebases. Documentation that needs to be updated for an upcoming release should target the `next` branch. Please change the target branch of this PR to `next` and rebase if needed.

**Comment by HarvsG** [comment]
> It seems that this PR is targeted against an incorrect branch since it has a parent PR on one of our codebases. Documentation that needs to be updated for an upcoming release should target the `next` branch. Please change the target branch of this PR to `next` and rebase if needed.

Is appropriate for current as changes still apply and are correct with the current version of HA. 
1. `prob_given_false` is already marked as required in the documentation
2. Other changes are corrections of mistakes
3. `Parent` PR is merged

**Comment by HarvsG** [comment]
Also paging @JudoWill, you may be interested in the changes that are afoot in https://github.com/home-assistant/core/pull/67631

**Comment by JudoWill** [comment]
Oooo, an update to the docs for estimating `prob_given_true` and `prob_given_false` and getting rid of `prob_given_false = 1-prob_given_true` when undefined. This is my jam!

I'll give it a read and see if I have any nudges.

My git/github skill is a bit naive, how would I get my commits into this PR?

**Comment by HarvsG** [comment]
> Oooo, an update to the docs for estimating `prob_given_true` and `prob_given_false` and getting rid of `prob_given_false = 1-prob_given_true` when undefined. This is my jam!
> 
> I'll give it a read and see if I have any nudges.
> 
> My git/github skill is a bit naive, how would I get my commits into this PR?

Probably best to start a new PR once this one is merged.

Or otherwise go here https://github.com/home-assistant/home-assistant.io/pull/24276/files, click the expand icon in the top left and then click the floating blue + icon next to the lines you want to change and describe the changes, then I can implement them.

Or make code suggestions:
https://haacked.com/archive/2019/06/03/suggested-changes/

By the way @JudoWill I have made some changes to the code as well in the pull request I linked above so now if the `to_state` evaluates to FALSE it will also update the prior. 

So
```yml
prior: 0.58 # home roughly 14/24 hours per day
observations:
  - platform: state
    entity_id: sensor.tv
    prob_given_true: 0.33  #4 hours of TV per 10 hours I’m home
    prob_given_false: 0.054 #15 minutes of TV per 14 hours away, I don’t like zeros here
    to_state: on
  - platform: state
    entity_id: sensor.tv
    prob_given_true: 0.66 # 8/12 Hours off while home
    prob_given_false: 0.946 # 13.75/14 hours off while I’m away
    to_state: off # this is the “TRUE” state now
```
can become
```yml
prior: 0.58 # home roughly 14/24 hours per day
observations:
  - platform: state
    entity_id: sensor.tv
    prob_given_true: 0.33  #4 hours of TV per 10 hours I’m home
    prob_given_false: 0.054 #15 minutes of TV per 14 hours away, I don’t like zeros here
    to_state: on
```
When this change comes in, you may want to edit your post on the forum 😄 
https://community.home-assistant.io/t/how-bayes-sensors-work-from-a-statistics-professor-with-working-google-sheets/143177

Lets take this discussion to the community forum: https://community.home-assistant.io/t/the-future-of-the-bayesian-integration-thoughts-wanted/465476

**Comment by JudoWill** [comment]
@HarvsG, what happens when a `sensor` can have 3 (or more) options? Like a `person` that can be `home`, `not_home`, or at any number of areas. Or an `input_select`. Or is that behavior limited to `binary_sensor` inputs?

**Comment by HarvsG** [comment]
> @HarvsG, what happens when a `sensor` can have 3 (or more) options? Like a `person` that can be `home`, `not_home`, or at any number of areas. Or an `input_select`. Or is that behavior limited to `binary_sensor` inputs?

Behind the scenes that gets converted in a new `multi_state` platform. What that means is the old style of updating is preserved:- the prior will only be updated if the value resolves to `True`. This prevents the problems you are no doubt foreseeing.

**Comment by HarvsG** [comment]
Ready for merge. JudoWill and I can discuss other changes in another thread.

**Review by frenck — CHANGES_REQUESTED** [review]
This PR is based on and targeting the wrong branch. 

https://github.com/home-assistant/core/pull/67631 is part of 2022.10; however, the exiting `current` documentation (and thus branch) is for 2022.9.

This PR should be targeting the `next` branch. Please change the target branch and git rebase the PR if needed.

../Frenck

**Comment by HarvsG** [comment]
> This PR is based on and targeting the wrong branch.
> 
> [home-assistant/core#67631](https://github.com/home-assistant/core/pull/67631) is part of 2022.10; however, the exiting `current` documentation (and thus branch) is for 2022.9.
> 
> This PR should be targeting the `next` branch. Please change the target branch and git rebase the PR if needed.
> 
> ../Frenck

I see what you are saying but I respectfully disagree. 
The changes proposed in this PR do not require [home-assistant/core#67631](https://github.com/home-assistant/core/pull/67631) and simply build on/correct https://github.com/home-assistant/home-assistant.io/pull/21899

Said another way, even if home-assistant/core#67631 hadn't been merged, these changes would still be applicable. (They are essentially just typos and corrections). I have updated the proposal text to make this clearer.

Also it keeps getting assigned to me (I think because I am the code owner).

**Comment by frenck** [comment]
> The changes proposed in this PR do not require

Your provided PR description is linked against a PR in core. While the PR is now edited, it even mentioned:

> Since merge of https://github.com/home-assistant/core/pull/67631 prob_given_false is now required.

"I see what you are saying but I respectfully disagree.", the only one you are disagreeing with is yourself at that point :)

**Comment by frenck** [comment]
> Also it keeps getting assigned to me (I think because I am the code owner).

That is correct and expected behavior.

**Review by frenck — APPROVED** [review]
Thanks, @HarvsG 👍

**Pull request merged**
